modules/flann/include/opencv2/flann/flann_base.hpp

```diff
@@ -45,6 +45,32 @@
 
 namespace cvflann
 {
+class FILEScopeGuard {
+
+public:
+    FILEScopeGuard() {};
```

Comment:
Remove default unused ctor.

(at least it doesn't initialize `file_` member and destructor calls `fclose()` unconditionally - sporadic WTF behavior with closing of non-owned files)